bench/tx-generator/src/Cardano/Benchmarking/CliArgsScript.hs

```diff
@@ -62,6 +62,7 @@ plainOldCliScript _ _ (FundsUtxo _ _ _) _ = error "plainOldCliScript FundsUtxo n
 plainOldCliScript _ _ (FundsSplitUtxo _ _) _ = error "plainOldCliScript FundsSplitUtxo not supported"
 plainOldCliScript cliPartialBenchmark benchmarkEra (FundsGenesis keyFile) (tracers, dslSet) = do
   case benchmarkEra of
+      AnyCardanoEra AlonzoEra  -> error "AlonzoEra not supported"
```

Comment:
Pretty sure the rest of the node (on `master`) already supports `Alonzo`, so we should fix this.

Reply:
Yeah, or some explanation of why we don't support alonzo era and what impact that might have.

Reply:
We do not want to lose all the history of the tx-generator. This PR is (feature wise) the latest version of tx-genrator
when we stopped development in the old repo.
After it is merged to `cardano-node` we plan to remove some of the features and `CliArgsScript.hs` will be removed all together.

Reply:
> We do not want to lose all the history of the tx-generator. 

What history?

The `master` branch of the node currently supports `Alonzo`. If this PR is merged it will almost certainly *break* it .

> After it is merged to cardano-node we plan to remove some of the features and CliArgsScript.hs will be removed all together.

No, you need to fix it first, then it can be merged.

Reply:
@erikd, that support is being worked on in an follow-up branch, and it's a non-trivial bit.

Reply:
Moreover, you are looking at the CLI-based configuration for the generator, which is only needed for a very short interim period, and will be removed quite soon.

Reply:
And yet still, the generator can be used with Alonzo clusters -- it's just that it can't generate Alonzo-specific transactions yet.

Reply:
So when are the commits going to be squashed?

Reply:
Serge did a first squash Yesterday.
We will squash it again, after the reviews, when it is ready to merge.

Reply:
I would must rather review *after* the squash rather than before.